arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowData.java

```diff
@@ -237,35 +239,38 @@ public void writeArrowDataType(File file) throws IOException {
       vectorSchemaRoot.setRowCount(numRows);
       for (Field field : vectorSchemaRoot.getSchema().getFields()) {
         FieldVector vector = vectorSchemaRoot.getVector(field.getName());
-        switch (vector.getMinorType()) {
-        case TINYINT:
+        switch (field.getName()) {
+        case "tinyIntField":
```

Comment:
This is a necessary change? What is the difference between decimalField2 and decimalField?

Reply:
Required, the scale of decimalfield and decimalfiled2 are different, one is 2, the other is 3, the main purpose is to capture the accuracy of whether it is effective